diesel_tests/tests/sqlite_specific_schema.rs

```diff
@@ -0,0 +1,64 @@
+use diesel::*;
+use super::User;
+
+//FIXME: We can go back to `infer_schema!` when codegen becomes generic
+table! {
+    users {
+        id -> Integer,
```

Comment:
Serial is just an alias right now. If we ever turned it back into a proper type, it would be invalid on SQLite as it's the PG specific name for an auto incrementing integer.